requirements.txt

```diff
@@ -17,8 +20,37 @@ tensorflow-model-remediation==0.1.7.1
 tensorflow-recommenders==0.7.2
 keras_cv==0.5.1
 keras_nlp==0.5.1
-fire==0.6.0
+jax==0.4.30
 
+# MLOps
+kubernetes==17.17.0
```

Comment:
Added `kubernetes` explicitly so the installation process doesn't need to install multiple versions and check compatibility.